lib/pages/detail/detail_content.dart

```diff
@@ -239,4 +146,47 @@ extension on Company {
       return null;
     }
   }
+
+  CompanyScoreData? scoreData() {
+    final int? plCapital = this.plCapital;
+    final int? plWorkers = this.plWorkers;
+    final int? plRnD = this.plRnD;
+    final int? plRegistered = this.plRegistered;
+    final int? plNotGlobEnt = this.plNotGlobEnt;
+    final int? plScore = this.plScore;
+
+    if (plCapital != null &&
+        plWorkers != null &&
+        plRnD != null &&
+        plRegistered != null &&
+        plNotGlobEnt != null &&
+        plScore != null) {
+      return CompanyScoreData(
+          plCapital: plCapital.toDouble(),
+          plWorkers: plWorkers != 0,
+          plRnD: plRnD != 0,
+          plRegistered: plRegistered != 0,
+          plNotGlobEnt: plNotGlobEnt != 0,
+          plScore: plScore);
+    }
+  }
+}
+
+class _ScoreSection extends StatelessWidget {
+  final Company company;
+
+  const _ScoreSection({super.key, required this.company});
+
+  @override
+  Widget build(BuildContext context) {
+    final scoreData = company.scoreData();
+
+     if (scoreData != null) {
+      if (scoreData.plScore == 0) {
+      }
+      return CompanyScoreWidget(data: scoreData);
+    } else {
+      return Container();
```

Comment:
Jak zrobisz poprzednie komentarze, to wystarczy, że tu zamiast `Container();` dasz `NoScoreMessage();` i zadanie będzie zrobione.